src/components/Project/Releases/index.js

```diff
@@ -597,29 +597,36 @@ export default class Releases extends React.Component {
   stopReleaseButton(release) {
     // ADB 8/21/18
     // Temporarily diasbling cancel/'stop release' button until it works
```

Comment:
you can delete this comment.

Reply:
i mean, it is still disabled for most users